docs/source/tutorials/registry_import.md

```diff
@@ -0,0 +1,114 @@
+        "-1",
+        sample.r1.local_path,
+        "-2",
+        sample.r2.local_path,
+        "-S",
+        str(sam_file),
+    ]
+
+    ...
+
+    output_location = f"latch:///Assembly Outputs/{sample.name}/covid_assembly.sam"
+
+    return LatchFile(str(sam_file), output_location)
+```
+
+Next, we can define a workflow that takes in a list of `dataclass`es. The workflow will use the [`map_task` construct](../basics/map_task.md) in the SDK to parallelize the `assembly_task` across a list of inputs.
+
+```python
+@workflow(metadata)
+def assemble_and_sort(samples: List[Sample]) -> List[LatchFile]:
+    return map_task(assembly_task)(sample=samples)
+```
+
+Now that we have set up the workflow logic, we can customize the workflow interface to display a sample sheet. To do so, we can set the `samplesheet` flag of `LatchParameter` equal to `True`.
+
+```python
+"""The metadata included here will be injected into your interface."""
+metadata = LatchMetadata(
+    display_name="Assemble FastQ Files (Registry Sample Sheet Version)",
+    documentation="your-docs.dev",
+    author=LatchAuthor(
+        name="Author",
+        email="author@gmail.com",
+        github="github.com/author",
+    ),
+    repository="https://github.com/your-repo",
+    license="MIT",
+    parameters={
+        "samples": LatchParameter(
+            display_name="Sample sheet",
+            samplesheet=True, # <======= flag to display sample sheet component
+            description="A list of samples and their sequencing reads",
+        )
+    },
+    tags=[],
+)
+```
+
+To preview what the workflow interface looks like, you can type:
+```bash
+latch preview <path_to_workflow_directory>
+```
+
+The command will open up a new page in the browser that displays a preview of the sample sheet component.
+
+![A preview of the sample sheet component on the workflow GUI](../assets/registry/samplesheet.png)
+
+When you click "Import from Registry", a new import modal will pop up. 
+
+![A preview of the sample sheet component on the workflow GUI](../assets/registry/import.png)
+
+Here, you can select the table of interest and samples to be used in the workflow.
+
+![A preview of the sample sheet component on the workflow GUI](../assets/registry/sample-selection.png)
+
+**Important Note**:
+
+* The types of columns in a Registry table must match one-to-one with the Python types of the data class defined in SDK. 
+* For example, the Registry table in the picture above has three columns: **Name**, **r1**, and **r2**, which have the types **Text**, **File**, and **File** in Registry, respectively. Similarly, the property `name`, `r1`, and `r2` in the data class `Sample` has the Python types `str`, `LatchFile`, and `LatchFile`.
+
+![A preview of the sample sheet component on the workflow GUI](../assets/registry/match-columns.png)
+
+To learn more about how to create columns with specific types in Registry, visit our [Registry Wiki here](https://latch.wiki/create-a-table).
```

Comment:
```suggestion
[See the wiki section on Registry types](https://latch.wiki/create-a-table) for more information.
```
kinda bad, needs to be much more specific
but that would need its own page probably

it also needs to explain how types match with Registry API types

Reply:
yeah I agree - I will make a new page. can we merge this for now so I can start sending to customers?